cms/templates/index.html

```diff
@@ -495,6 +523,74 @@ <h4 class="title">${_('Your Course Creator Request Status:')}</h4>
       </div>
       % endif
 
+
+      %if len(courses) > 0 or optimization_enabled:
+      <div class="archive archive-tab">
+        <ul class="list-courses">
+          %for course_info in sorted(courses, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
+          % if course_info['archived'] == True:
```

Comment:
same as above - no need for == here, you can just write 'if course_info['archived']'